saleor/webhook/event_types.py

```diff
@@ -65,6 +66,8 @@ class WebhookEventType:
     PAYMENT_CONFIRM = "payment_confirm"
     PAYMENT_PROCESS = "payment_process"
 
+    SHIPPING_LIST_METHODS = "shipping_list_methods"
```

Comment:
As we should also have a webhook for the order shipping method, I would suggest renaming this webhook to
`SHIPPING_LIST_METHODS_FOR_CHECKOUT` and `SHIPPING_LIST_METHODS_FOR_ORDER`. We want to have a possibility to integrate shipping providers only for checkout or only for order OR for both. In current way we will have to use a single webhook to handle two different types, which will complicate the logic as we don't do that in any place.